tests/plugin/text/intent_entity_mutator/test_mutation_cases.yaml

```diff
@@ -0,0 +1,259 @@
+- intent_name: "entity_only"
+  current_state: "ASK_CALLBACK_TIME"
+  transcript: "I will pay tomorrow"
```

Comment:
What is this file for? i already saw a test cases file?

Reply:
This is the test cases file. Another yaml is the rules file.